bin/anycabled

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+require "anycable/cli"
+require "daemons"
+
+options = {
+  dir: "tmp/pids",
+  log_output: false
+}
+
+# Preserve current directory. We need it inside the server.
+current_dir = Dir.pwd
+
+Daemons.run_proc("anycable", options) do
+  Dir.chdir current_dir
+  AnyCable::CLI.new.run
```

Comment:
> anycable cli args are not handled

Looks like we can pass args to the process: https://github.com/thuehlinger/daemons/blob/master/lib/daemons.rb#L88

Reply:
And this a most popular gem for daemonizing. Documentations is misleading. They handle ARGV for[ external scripts only](https://github.com/thuehlinger/daemons/blob/master/lib/daemons/application.rb#L223-L224). Such a shame. Anyway, it won't work [without bounded proc execution]( myproc.call).

Reply:
Implemented in 43eb8a3